### DISCUSSION TITLE
text-index what's the prefix that is automatically determined

### DISCUSSION THREAD

**Discussion by alpapan** [GMOD/jbrowse-components#4200] [General] [answered]
How can we know what is the size of the prefix?

Docs read it is automatically determined. They also read that it should be increased if many elements/genes have similar names but we have no idea what to increase it from.

I also [note that different jb version use different prefix defaults](https://github.com/GMOD/jbrowse-components/discussions/3055) (but docs say it is automatically determined, presumably after this [post](https://github.com/GMOD/jbrowse-components/discussions/2803))

Would be nice if it the size was stored in the `_meta.json`.

**Comment by cmdcolin** [comment, marked as answer]
this is a good point, will see if this can be outputted to meta. right now, looking directly at the .ixx file would be required
